src/sentry/tasks/commit_context.py

```diff
@@ -92,12 +92,17 @@ def queue_comment_task_if_needed(
 
     merge_commit_sha = response[0]["merge_commit_sha"]
 
-    pr_query = PullRequest.objects.filter(
-        organization_id=commit.organization_id,
-        repository_id=commit.repository_id,
-        merge_commit_sha=merge_commit_sha,
+    # get first PR that merged the commit
+    pr = (
+        PullRequest.objects.filter(
+            organization_id=commit.organization_id,
+            repository_id=commit.repository_id,
+            merge_commit_sha=merge_commit_sha,
+        )
+        .order_by("date_added")
```

Comment:
Is this column and the one's we're querying by indexed? We might run into performance problems and affect the DB with this query if the table is big

Reply:
we have this index? https://github.com/getsentry/sentry/blob/df0975742f5fed62ecb994af485004ec3cb827af/src/sentry/models/pullrequest.py#L78